bin/runex

```diff
@@ -22,4 +22,4 @@ if command -v accumulo > /dev/null 2>&1 ; then
   av_arg="-Daccumulo.version=`accumulo version`"
 fi
 
-mvn -q exec:java -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args"
+mvn -q exec:java -Dexec.mainClass="org.apache.accumulo.examples.$main_class" $av_arg -Dexec.args="$main_args" -Dlog4j.configuration=file:"./src/main/resources/log4j.properties"
```

Comment:
Moot point now... compiling with it in src/main/resources puts it on the classpath.